leshan-core/src/main/java/org/eclipse/leshan/core/request/BindingMode.java

```diff
@@ -36,5 +37,12 @@
     US,
```

Comment:
I know this is out of scope of this PR (maybe even irrelevant), it would be nice to try and use EnumSet instead of specific enum per combination. As more Transport layer are added (in the future), this will become hard to manage.